### PR TITLE
Adds final draft of ANS Abstract 

### LINKED ISSUE #65: ANS abstract for winter meeting
This issue can be closed when an abstract for the ANS Winter meeting has been uploaded and submitted to ANS.

### PR DESCRIPTION
**Pull request by samgdotson**
This PR adds the final draft of the abstract I submitted for the ANS Annual Meeting

### CODE REVIEW

File: publications/abstracts/reservoir-computing-abs/background.tex [hunk 1]
```diff
@@ -1,44 +1,49 @@
...
+for decarbonization \cite{dotson_optimal_2020}. Due to the degree of wind
+penetration, the University is sometimes
 forced to sell electricity back to the grid operator, MISO, at a loss because
 of overproduction
 from wind energy. Thus, a reliable prediction of electricity production from
 wind and other variable sources will reduce the likelihood of these events.
+Figure \ref{figure:vre} shows the total demand and net demand profiles at the
+\gls{uiuc}.
+Daily variations in electricity demand are reasonably predictable, with a
+usual evening minimum around 40 MW. Weekdays have peaks near 50MW and are
```
Comment (datw0258) [inline publications/abstracts/reservoir-computing-abs/background.tex]: Do you want to have a space between the number and unit? I think either could work (50MW or 50 MW) as long as you are consistent.

File: publications/abstracts/reservoir-computing-abs/background.tex [hunk 2]
```diff
@@ -1,44 +1,49 @@
...
+  \label{figure:vre}
+\end{figure}
 
 \glspl{ESN}, a flavor of reservoir computing, are a modern
-machine learning algorithm that enables accurate short
-to medium term predictions. Pathak et. al used an \gls{ESN} to predict the
-evolution of a chaotic system, a laminar flame front, up to seven Lyapunov
+machine learning algorithm that can learn the dynamics of a chaotic system.
+Pathak et. al used an \gls{ESN} to predict the
+evolution a laminar flame front up to seven Lyapunov
```
Comment (datw0258) [inline publications/abstracts/reservoir-computing-abs/background.tex]: Should it read "...evolution **of** a laminar flame front..." ?

**Review by datw0258 — CHANGES_REQUESTED** [review]
Looks good! Just a few minor comments

**Comment by samgdotson** [comment]
@datw0258 could you go ahead and merge this. The deadline is long passed.

**Pull request merged**